### PR TITLE
extending SplitOlderIpadSwitch to give us time to discuss core taking over all the ipads

### PR DESCRIPTION
**Pull request by michaelwmcnamara**
Eventually we will probably extend core to all the ipads but this needs to be discussed and done right - so for now just extending the switch

### CODE REVIEW

**Comment by janua** [comment]
:+1:

File: common/app/conf/switches/FeatureSwitches.scala [hunk 1]
```diff
@@ -292,7 +292,7 @@ trait FeatureSwitches {
     "ipad-split-capabilities",
     "If switched on then this gives older ipads the stripped down front but full articles",
     safeState = On,
-    sellByDate = new LocalDate(2015, 9, 30),
+    sellByDate = new LocalDate(2015, 11, 30),
```
Comment (JustinPinner) [inline common/app/conf/switches/FeatureSwitches.scala]: November 30th is a Monday. Will this revert effectively on Sunday night? Do we care about that?

**Comment by michaelwmcnamara** [comment]
Thinking is that we will have committed or decided against core for ipads by that date, but better safe than sorry :-)

**Comment by johnduffell** [comment]
:speedboat: this switch is well overdue so thanks for fixing :+1:

**Comment by johnduffell** [comment]
ship it!!

**Pull request merged**